mobile_verifier/src/reward_shares.rs

```diff
@@ -1589,15 +1605,17 @@ mod test {
                 hotspot_key: gw1.clone(),
                 cell_type: CellType::NovaGenericWifiIndoor,
                 coverage_object: g1_cov_obj,
-                location_trust_score_multiplier: dec!(0.25),
+                distances_to_asserted: Some(vec![0]),
```

Comment:
Also, it's entirely possible that we set the trust score to 0.25 for another reason besides the distance to asserted: it could be 0.25 because the distance to a coverage object hex is too large